src/probability/borelScript.sml

```diff
@@ -9293,6 +9293,207 @@ Proof
  >> rw [IN_MEASURABLE_BOREL_BOREL_ABS_POWR]
 QED
 
+(***********************)
+(*   Further Results   *)
+(***********************)
+
+(*  Some of these do require addition simplifier manipulations.
+      I try to have the simplifier modified for as little as possible in order to make
+      it easier to relocate things to more appropriate places above.
+      - Jared Yeager                                                                   *)
+
+(*** IN_MEASURABLE_BOREL Theorems ***)
+
+Theorem IN_MEASURABLE_BOREL_CONG:
```

Comment:
Please make this an iff oriented from f to g. That way it satisfies the constraints to be a Definition congruence, used in termination analysis.

Reply:
I'm happy to (especially since that form is redundant with _EQ'), but I'll need you to elaborate on what exactly that entails. I'm still very fuzzy on congruences. Would that just be:
∀a f g. (∀x. x ∈ space a ⇒ f x = g x) ⇒ (f ∈ Borel_measurable a ⇔ g ∈ Borel_measurable a)

Reply:
Similar changes can be done also for `IN_MEASURABLE_CONG` in `sigma_algebraTheory`.